content/index.html

```diff
@@ -52,8 +53,8 @@
 		<!-- End Google Analytics -->
 
 	</head>
-	<body style="height:100%">
-		<nav class="navbar navbar-default" ng-controller="navController">
+	<body ng-controller="navController" ng-style="isPublicProfile() === true ? {'background': '#f5f8fa'} : {'background': 'white'}">
```

Comment:
`=== true` is redundant, should be
` ng-style="isPublicProfile() ? {'background': '#f5f8fa'} : {'background': 'white'}"`